Rocket.Chat/API/Clients/UploadClient.swift

```diff
@@ -14,8 +14,8 @@ struct UploadClient: APIClient {
         self.api = api
     }
 
-    func upload(roomId: String, data: Data, filename: String, mimetype: String, description: String, completion: (() -> Void)? = nil, versionFallback: (() -> Void)? = nil) {
-        let req = UploadRequest(
+    func uploadMessage(roomId: String, data: Data, filename: String, mimetype: String, description: String, completion: (() -> Void)? = nil, versionFallback: (() -> Void)? = nil) {
```

Comment:
@RocketChat/ios since this function is about uploading a media message to a room I thought it would be better to call it `uploadMessage` instead of just `upload`. Let me know if you guys think we should back to `upload`.

Reply:
Yes, much better now, specially since we have two upload methods :-)